paimon-spark/paimon-spark-common/src/main/java/org/apache/paimon/spark/SparkCatalog.java

```diff
@@ -390,7 +390,7 @@ private static SchemaChange.Move getMove(
 
     private Schema toInitialSchema(
             StructType schema, Transform[] partitions, Map<String, String> properties) {
-        Map<String, String> normalizedProperties = mergeSQLConf(properties);
+        Map<String, String> normalizedProperties = mergeSQLConf(properties, false);
```

Comment:
We can just remove this line without modifying mergeSQLConf

Reply:
good idea~